test/extensions/filters/network/redis_proxy/config_test.cc

```diff
@@ -34,9 +34,6 @@ TEST(RedisProxyFilterConfigFactoryTest, NoUpstreamDefined) {
 
   NiceMock<Server::Configuration::MockFactoryContext> context;
 
-  EXPECT_THROW_WITH_MESSAGE(
```

Comment:
Needs an EXPECT of some kind for the test.

Reply:
Done. It was Local build issue